### PR TITLE
Change admin auth

### PR DESCRIPTION
**Pull request by mcalmer**
Adapted the authentication chapter of the admin guide.

### CODE REVIEW

**Comment by Loquacity** [comment]
Added my edit.

File: modules/administration/pages/auth-methods.adoc [hunk 1]
```diff
@@ -1,115 +1,66 @@
...
+. Check the [guimenu]``Pluggable Authentication Modules (PAM)`` checkbox.
+It is below the password and password confirmation fields.
 
-You can now create users with the same ID used by eDirectory, and check the [guimenu]``Use PAM`` checkbox in the {productname} {webui}.
 
 
+[IMPORTANT]
+====
+Changing the password in the {productname} {webui} changes only the local password on the {productname} Server.
+If PAM is enabled for that user, this password might not be used at all.
```
Comment (keichwa) [inline modules/administration/pages/auth-methods.adoc]:
```suggestion
If PAM is enabled for that user, the local password might not be used at all.
```
for clarity

**Comment by Loquacity** [comment]
@mcalmer do you need one of us to merge?

**Comment by Loquacity** [comment]
:+1:

**Pull request merged**